bridge/irc_listener.go

```diff
@@ -39,6 +40,9 @@ func newIRCListener(dib *Bridge, webIRCPass string) *ircListener {
 		listener.JoinChannels()
 	})
 
+	listener.AddCallback("NICK", listener.nickTrackNick)
+	listener.AddCallback("QUIT", listener.nickTrackQuit)
```

Comment:
Don't these need to be added to the map too?

Reply:
These 2 are for `puppetNicks` related things and need to be permanently enabled.

Reply:
ah, cool 👍